src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -22,10 +22,14 @@ def update
                       fetch_notifications.where(id: params[:notification_ids])
                     end
     # rubocop:disable Rails/SkipsModelValidations
-    unless notifications.update_all('delivered = !delivered')
+    updated_notifications = notifications.update_all('delivered = !delivered')
+    # rubocop:enable Rails/SkipsModelValidations
+
+    if updated_notifications.zero?
       flash.now[:error] = "Couldn't mark the notifications as #{notifications.first.unread? ? 'read' : 'unread'}"
+    else
+      send_notifications_information_rabbitmq(updated_notifications.size)
```

Comment:
`updated_notifications` is already an integer, `.size` isn't needed.

```suggestion
      send_notifications_information_rabbitmq(updated_notifications)
```

Reply:
yup, now I see, I was expecting the the array of elements changed, not a number. :+1: